e2-studio-star-rx72n-firmware/build-wsl.sh

```
@@ -0,0 +1,120 @@
+# Convert ALL Windows paths to WSL paths
+# Step 1: Convert C:/ to /mnt/c/ (forward slash variant)
+sed -i 's|C:/|/mnt/c/|g' "$TEMP_MAKEFILE"
+
+# Step 2: Convert C:\ to /mnt/c/ (backslash variant - must come before general backslash replacement)
+sed -i 's|C:\\\\|/mnt/c/|g' "$TEMP_MAKEFILE"
+
+# Step 3: Fix remaining escaped backslashes in paths (\\)
+sed -i 's|\\\\|/|g' "$TEMP_MAKEFILE"
+
+# Step 4: Replace Windows GNURX library paths with WSL GNURX paths
+sed -i 's|-L"/mnt/c/ProgramData/GCC for Renesas RX [^"]*"|-L"${HOME}/gnurx/rx-elf/lib/gcc/rx-elf/14.2.0.202511-GNURX" -L"${HOME}/gnurx/rx-elf/rx-elf/lib"|g' "$TEMP_MAKEFILE"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)